hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleUtility.java

```diff
@@ -333,8 +327,9 @@ private static void addToHash(final Hasher hasher, final Key keyToAdd) {
     }
 
     @SuppressWarnings("UnstableApiUsage")
-    private static void addToHash(final Hasher hasher, final Timestamp timeToAdd) {
-        final byte[] bytes = Timestamp.PROTOBUF.toBytes(timeToAdd).toByteArray();
+    private static void addToHash(final Hasher hasher, final Long timeToAdd) {
```

Comment:
Actually, this method can be entirely removed.

Reply:
This method is needed until the ProtoLong -> putLong change is implemented as described above